egress/egressonlytracking.yaml

```diff
@@ -70,7 +75,42 @@ components:
           format: uint32
           default: 1
           minimum: 1
-          x-field-uid: 3
+          x-field-uid: 4
+
+    EgressOnlyTracking.TxOffset:
+      description: >-
+        A container of Tx offset properties.
+      type: object
+      properties:
+        choice:
+          description: >-
+            Choose "auto" when optional Tx statistics in egress only tracking are required while fetching egress only stats by opting "tx_metric" in get_metrics - egress_only_tracking - tagged_metric - metric_names.  or both Tx and Rx offset of tracked field in packet is the same. Otherwise choose "custom" when the Tx and Rx offsets are different due to DUT adding/ modifying or deleting encapsulation protocol header e.g. when the egress-only tracked packets are MACsec encapulated and the Tx and Rx side have different configuration such that the MACSec header is added/modified or removed.
```

Comment:
Choose "auto" when the Tx and Rx offsets are same i.e. DUT has not modified the packet contents upto the field being tracked. 
Choose "custom" when Tx and Rx offsets are different due to DUT adding/ modifying or deleting encapsulation protocol header e.g. when the egress-only tracked packets are MACsec encapulated and the Tx and Rx side have different configuration such that the MACSec header is added/modified or removed.